LevelDBStore/LevelDBStore.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>3.0.0-CI00810</Version>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RootNamespace>Neo</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Update="LevelDBStore\config.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Neo" Version="3.0.0-CI00810" />
```

Comment:
Time to move levelDB to a nuget in order to avoid library issues? (maybe hosted by neo)

Reply:
What library issues?

Reply:
If you compile from the source, you need the levelDB dll or you can't run the node, with release there is no problem

Reply:
Currently only windows needs it.

Reply:
Ok, we can improve it later !